src/text/analyze.rs

```diff
@@ -41,29 +41,46 @@ impl<I> Analyze<I> {
     pub fn needs_bidi_resolution(&self) -> bool {
         self.state.needs_bidi
     }
+
+    /// Sets the word breaking strength that will be used to analyze the next character.
+    pub fn set_break_strength(&mut self, strength: WordBreakStrength) {
+        self.state.strength = strength;
+    }
+}
+
+/// Word breaking strength (corresponds to <https://drafts.csswg.org/css-text/#word-break-property>).
+#[derive(Copy, Clone, PartialEq, Eq, PartialOrd, Ord, Hash, Debug)]
+pub enum WordBreakStrength {
+    Normal,
```

Comment:
Might be reasonable to make it defaultable.
```suggestion
/// Word breaking strength (corresponds to <https://drafts.csswg.org/css-text/#word-break-property>).
#[derive(Copy, Clone, PartialEq, Eq, PartialOrd, Ord, Hash, Debug, Default)]
pub enum WordBreakStrength {
    #[default]
    Normal,
```

Reply:
Might also make sense to make it `repr(u8)` as it is in your Parley PR, so that you can get rid of the conversion altogether there.